packages/fluentui/react-northstar/src/themes/teams/siteVariables.ts

```diff
@@ -53,6 +54,7 @@ export const lineHeightSmall = 1.3333;
 export const lineHeightMedium = 1.4286;
 export const lineHeightLarge = 1.3333;
 export const lineHeightLarger = 1.3333;
+export const lineHeightLargest = 1.3333;
```

Comment:
Why is the line-height not changing? Do these values come from design?

Reply:
There's no line height defined for these sizes.